File: ImplementstrStr/ImplementstrStr.py
# Runtime: 24 ms, faster than 96.55% of Python3 online submissions for Implement strStr().
# Memory Usage: 14 MB, less than 57.30% of Python3 online submissions for Implement strStr().

class Solution:
    def strStr(self, haystack, needle):
        if needle in haystack:
            return haystack.index(needle)
        else:
            return -1

File: Easy/ImplementstrStr/ImplementstrStr.cpp
/*
Runtime: 8 ms, faster than 67.22% of C++ online submissions for Implement strStr().
Memory Usage: 6.8 MB, less than 71.86% of C++ online submissions for Implement strStr().
*/

class Solution {
public:
    int strStr(string haystack, string needle) {
        if (haystack.find(needle) != std::string::npos) {
            return haystack.find(needle);
        } else {
            return -1;
        }
    }
};